src/main/resources/EsFieldsmapping/discussionEsMapping.json

```diff
@@ -36,5 +39,11 @@
       "yyyy-MM-dd'T'HH:mm:ss.SSS'Z'",
       "epoch_millis"
     ]
+  },
+  "answerPosts": {
+    "type": "keyword"
+  },
+  "media": {
```

Comment:
this suppose to be mediaUrls -- as we store only urls

Reply:
removed